packages/react-core/src/components/Modal/ModalContent.tsx

```diff
@@ -63,8 +69,11 @@ export const ModalContent: React.FunctionComponent<ModalContentProps> = ({
   onClose = () => undefined as any,
   variant = 'default',
   width = -1,
-  modalBoxAriaDescribedById = '',
-  id = '',
+  'aria-describedby': ariaDescribedby = '',
+  'aria-labelledby': ariaLabelledby = '',
+  boxId = '',
+  labelId = '',
+  descriptorId = '',
```

Comment:
You can destructure `id` (even though it's unused) to avoid passing it to ModalBoxBody. Add an `// eslint-ignore-next-line @typescript-eslint/no-unused-vars` beforehand to quiet the linter.